docs/guide/features.md

```diff
@@ -48,6 +48,12 @@ Vite's default types are for its Node.js API. To shim the environment of client
 }
 ```
 
+Also, you can add a `d.ts` declaration file:
```

Comment:
You may swap this section with the "types" above, cause this PR made it to the new default :)

Reply:
Done